src/test/streaming_test.cpp

```diff
@@ -449,6 +449,256 @@ node {
     ASSERT_EQ(executor.inferStream(this->firstRequest, this->stream), StatusCode::OK);
 }
 
+// Generative AI case + automatic timestamping server-side - Python
+
+#include <pybind11/embed.h>  // everything needed for embedding
+namespace py = pybind11;
+
+#include "../mediapipe_internal/pythonnoderesource.hpp"
+#include "../python/python_backend.hpp"
+
+static std::unordered_map<std::string, std::shared_ptr<PythonNodeResource>> prepareInputSidePacket(const std::string& handlerPath, PythonBackend* pythonBackend) {
+    // Create side packets
+    auto fsHandlerPath = std::filesystem::path(handlerPath);
+    fsHandlerPath.replace_extension();
+
+    std::string parentPath = fsHandlerPath.parent_path();
+    std::string filename = fsHandlerPath.filename();
+
+    py::gil_scoped_acquire acquire;
+    py::module_ sys = py::module_::import("sys");
+    sys.attr("path").attr("append")(parentPath.c_str());
+    py::module_ script = py::module_::import(filename.c_str());
+    py::object OvmsPythonModel = script.attr("OvmsPythonModel");
+    py::object pythonModel = OvmsPythonModel();
+
+    std::shared_ptr<PythonNodeResource> nodeResource = std::make_shared<PythonNodeResource>(pythonBackend);
+    nodeResource->nodeResourceObject = std::make_unique<py::object>(pythonModel);
+
+    std::unordered_map<std::string, std::shared_ptr<PythonNodeResource>> nodesResources{{"pythonNode", nodeResource}};
+    return nodesResources;
+}
+
+TEST_F(StreamingTest, SingleStreamSend1Receive3Python) {
+    py::scoped_interpreter guard{};  // start the interpreter and keep it alive
```

Comment:
Adding any python code in the test does not ensure us that OVMS does it correctly outside of unit tests

Reply:
I'll try to change it after rebase when applying suggestion from the previous comment.